src/main/java/edu/kis/vh/nursery/list/IntLinkedList.java

```diff
@@ -2,15 +2,15 @@
 
 public class IntLinkedList {
 
-    Node last;
-    int i;
+    private Node last;
+    int counter;
 
     public void push(int i) {
         if (last == null)
             last = new Node(i);
         else {
             last.next = new Node(i);
-            last.next.prev = last;
+            last.next.previous = last;
             last = last.next;
```

Comment:
use getters and setters (Lack of encapsulation in Node class (Task 11 in "accessory i hermetyzacja"))